code/modules/shuttle/cas_plane.dm

```diff
@@ -21,11 +21,14 @@
 	var/mob/living/carbon/human/occupant
 	///Animated cockpit /image overlay, 96x96
 	var/image/cockpit
+	/// Whether CAS is usable or not.
+	var/cas_usable = 0
```

Comment:
This doesn't need to be defined, since that makes it null anyway. Otherwise it can be FALSE

Reply:
Gotcha.